src/main/scala/org/apache/spark/sql/internal/oap/OapConf.scala

```diff
@@ -27,6 +27,13 @@ import org.apache.spark.sql.oap.adapter.SqlConfAdapter
 
 object OapConf {
 
+  val OAP_ORC_ENABLED =
+    SqlConfAdapter.buildConf("spark.sql.oap.orc.enable")
+      .internal()
+      .doc("Whether enable oap file format when encounter parquet files")
```

Comment:
.doc("Whether enable oap file format when encounter parquet files")
needs to change accordingly. It has "parquet files" in the text.

Reply:
Sure. No problem.